minio/minio
# Allow folders inside buckets to be opened in a new tab

## Description

Check if a list item is a folder and if so change `href` attribute inside the `a` tag to lead to inside the folder.

## Motivation and Context


Fixes #7836

## Regression


No

## How Has This Been Tested?



1. Create folders inside buckets and inside folders to test the scenarios
2. Right click on a folder and verify that it can be opened on a new tab
3. Click on a folder and verify that it is opened
4. Try clicking on list items that aren't folders to verify that the previous behavior is unchanged

## Types of changes


## Checklist:
